Import css-scrollbars WPT tests

lukewarlow opened the pull request:
#### 9a31af76ac06099ee49118dd9c7c7b578ac12d3b
<pre>
Import css-scrollbars WPT tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=256792">https://bugs.webkit.org/show_bug.cgi?id=256792</a>

Reviewed by Tim Nguyen.

WPT at cb1e54222a206b4cbb5211f7e24878a267b820f1

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/META.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/auto-scrollbar-inline-children-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/auto-scrollbar-inline-children.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/inheritance-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/inheritance.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/multicol-in-orthogonal-writing-mode-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-003-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-004-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-005-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-006-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-007-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-008-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-keywords-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-keywords.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-parsing-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-parsing.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbars-chrome-bug-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbars-chrome-bug-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbars-chrome-bug-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/support/viewport-scrollbar-body-frame.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/support/viewport-scrollbar-frame.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/support/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/textarea-scrollbar-width-none-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/textarea-scrollbar-width-none-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/textarea-scrollbar-width-none.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/transparent-on-root-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/transparent-on-root-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/transparent-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/viewport-scrollbar-body-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/viewport-scrollbar-body-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/viewport-scrollbar-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/viewport-scrollbar-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/viewport-scrollbar-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/viewport-scrollbar.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/w3c-import.log: Added.
* LayoutTests/platform/ios-wk2/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/264178@main">https://commits.webkit.org/264178@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/186626e7a54cf3f8568f55e6ec4e9ad80b1d10b2

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/6874 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/7103 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/7287 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/8472 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/7121 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/8382 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/7043 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/10021 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/6995 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/7688 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/6323 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/8571 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/5069 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/6237 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/14030 "Passed tests") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/6754 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/6323 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/9118 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/6805 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/5571 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/6182 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/6151 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/10366 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/802 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/6560 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Looks good. Can you add the commit hash of the upstream commit you imported from in the commit message?

lukewarlow committed:
Import css-scrollbars WPT tests
https://bugs.webkit.org/show_bug.cgi?id=256792

Reviewed by Tim Nguyen.

WPT at cb1e54222a206b4cbb5211f7e24878a267b820f1

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/META.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/auto-scrollbar-inline-children-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/auto-scrollbar-inline-children.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/inheritance-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/inheritance.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/multicol-in-orthogonal-writing-mode-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-001-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-002-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-003-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-004-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-005-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-006-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-007-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-008-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-keywords-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-keywords.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-parsing-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-parsing.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbars-chrome-bug-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbars-chrome-bug-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbars-chrome-bug-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/support/viewport-scrollbar-body-frame.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/support/viewport-scrollbar-frame.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/support/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/textarea-scrollbar-width-none-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/textarea-scrollbar-width-none-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/textarea-scrollbar-width-none.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/transparent-on-root-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/transparent-on-root-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/transparent-on-root.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/viewport-scrollbar-body-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/viewport-scrollbar-body-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/viewport-scrollbar-body.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/viewport-scrollbar-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/viewport-scrollbar-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/viewport-scrollbar.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/w3c-import.log: Added.
* LayoutTests/platform/ios-wk2/TestExpectations:

Canonical link: https://commits.webkit.org/264178@main
